tools/xcms/.shed.yml

```diff
@@ -80,7 +80,7 @@ repositories:
       XCMS: http://www.bioconductor.org/packages/release/bioc/html/xcms.html 
     owner: lecorguille
     include:
-      [*default-includes, abims_xcms_refine.xml, xcms_refine.r]
+      [*default-includes, abims_xcms_retcor.xml, xcms_retcor.r]
```

Comment:
In general I would not use include or exclude directives. It's hard to keep them updated.

Reply:
Thanks for fixing